tools/cellxgene_census_builder/src/cellxgene_census_builder/build_soma/manifest.py

```diff
@@ -90,6 +90,7 @@ def load_manifest_from_CxG() -> list[Dataset]:
             collection_id=dataset["collection_id"],
             collection_name=null_to_empty_str(dataset["collection_name"]),
             collection_doi=null_to_empty_str(dataset["collection_doi"]),
+            collection_doi_label=null_to_empty_str(dataset.get("collection_doi_label")),
```

Comment:
Here you are using `dataset.get("...")` which does simply returns `None` if the key doesn't exist - that is, it does not raise an exception. The other calls use `dataset["..."]` which raise on exception if the key doesn't exist.

Should this semantics be made consistent? Maybe the other key lookups in `null_to_empty_str()` should use `dataset.get(...)` instead?